### PR TITLE
[Bug 1201656] Use correct product_key for FxiOS in AAQ.

### PR DESCRIPTION
**Pull request by mythmon**
Take two. This is a simpler, and means we didn't screw up as much. Yay!

@willkg r?

### CODE REVIEW

File: kitsune/questions/views.py [hunk 1]
```diff
@@ -518,7 +518,7 @@ def aaq(request, product_key=None, category_key=None, showform=False,
             if 'firefox' in ua and 'android' not in ua:
                 product_key = 'firefox-os'
             elif 'fxios' in ua:
-                product_key = 'fxios'
+                product_key = 'ios'
```
Comment (willkg) [inline kitsune/questions/views.py]: Typo.

Haha--kidding!

**Comment by willkg** [comment]
This looks good to me as far as I can tell. If Travis is cool with it, it's worth testing on -stage.

**Comment by mythmon** [comment]
This looks good on stage. Travis had an error in the build (is it time to ditch Travis now?). I restarted that job. Now more waiting.

**Comment by mythmon** [comment]
I got confirmation from jsavage and rolandtanglao about this on stage, and Travis is happy. Merging.

**Pull request merged**